src/modal/docs/readme.md

```diff
@@ -1,4 +1,4 @@
-`$modal` is a service to quickly create AngularJS-powered modal windows.
+The `$modal` is an AngularJS `service` (a `provider`) to quickly create AngularJS-powered modal windows.
```

Comment:
All great points! Angular makes it really sweet an easy by avoiding calling a Service two different things :) 

How about:
The `$modal` is an AngularJS Service registered as  `provider` ... ?